index.html

```diff
@@ -506,50 +509,51 @@
           input.oninput = function(event) {
             imageCapture.setOptions({zoom: event.target.value});
           }
+          input.hidden = false;
         })
         .catch(e => {
           console.error("Uh, oh, getPhotoCapabilities() failed: ", e);
         });
       }
 
       function takePhoto() {
-        imageCapturer.takePhoto()
-          .then((blob) => {
-            console.log("Photo taken: " + blob.type + ", " + blob.size + "B")
-
-            const imageTag = document.querySelector('input[type="img"]');
-            imageTag.src = URL.createObjectURL(blob);
-          })
-          .catch((err) => {
-            console.error("takePhoto() failed: ", e);
-          });
+        imageCapture.takePhoto().then(blob => {
```

Comment:
I'm trying to put the .then() in a new line,
indented by four spaces, like in l.488, 599 etc. 
Let's pick one and be consistent?